presto-main/src/main/java/com/facebook/presto/sql/planner/plan/Patterns.java

```diff
@@ -261,6 +253,14 @@ private Patterns() {}
         }
     }
 
+    public static class DistinctLimit
```

Comment:
this belongs in the previous commit where DistinctLimit was introduced.

Reply:
fixed